static/sass/_pattern_cve.scss

```diff
@@ -71,6 +74,16 @@
     position: relative;
   }
 
+  .cve-table .esm-info {
+    width: 100%;
```

Comment:
Linting - these should be in alphabetical order

Reply:
Fixed!